src/test/java/edu/ucla/library/prl/harvester/services/HarvestJobSchedulerServiceIT.java

```diff
@@ -63,6 +63,12 @@ public class HarvestJobSchedulerServiceIT {
     private static final Logger LOGGER =
             LoggerFactory.getLogger(HarvestJobSchedulerServiceIT.class, MessageCodes.BUNDLE);
 
+    private static final String DATABASE_INITIALIZED = "Database initialized";
```

Comment:
Is there a reason to not use resource bundle values here?

Reply:
Nope, I'll change that

Reply:
I guess it tends to give me pause because that file is in src/main/resources and our test classes are in src/test.

Reply:
Guess a separate resources file could be set up in src/test/resources and the plugin configured to render that, too, if we want to maintain that separation in logging resource files. It should be possible.